### ISSUE TITLE
Check Pronunciaton: missing contentDescription

### ISSUE THREAD

**Issue by david-allison** [ankidroid/Anki-Android#16320]
The `Reviewer - Check Pronunciation` and the `Note Editor - Record Audio` buttons are not usable by partially-sighted people, as the buttons lack `android:contentDecription`. Screen readers do not know what these images do.

https://developer.android.com/guide/topics/ui/accessibility/apps#describe-ui-element

Files:

- [ ] https://github.com/ankidroid/Anki-Android/blob/9c5ce7cc1e6a6cf5a6beb00a8bbe777ae080251d/AnkiDroid/src/main/res/layout/activity_audio_recording_reviewer.xml
- [ ] https://github.com/ankidroid/Anki-Android/blob/e54f8eb97049b00eebf8e761abde604b099bf902/AnkiDroid/src/main/res/layout/activity_audio_recording.xml

**Translations**
See our [Development Guide ](https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#adding-translations) for how to add translations.

TL;DR: Only add to `/values`, we'll take care of the rest after your PR is merged

**Testing**
Use https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en&gl=US

You do not need to fix all issues, as this is a https://github.com/ankidroid/Anki-Android/labels/Good%20First%20Issue%21, do as much as you feel confident with, and tell us if anything remains

**Comment by Aditya13s** [comment]
Assign to me. I am working on it.
